CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+### Bug Fixes
+- [crypto-org-chain/ethermint#19](https://github.com/crypto-org-chain/ethermint/pull/19) revert tharsis#786 because it contains consensus breaking changes
+
 *November 29, 2021*
 
 ## v0.6.3
```

Comment:
should it be removed from 0.6.3 or will this be a separate 0.6.4 release? it seems the 0.6.3 tag was removed?

Reply:
I originally intended to recreate v0.6.3 without the commit but since the commit specifying release v0.6.3 has already been already, it might be better to just do a separate v0.6.4 release. 

I removed 0.6.3 tag because this version should never be used (intentionally or unintentionally)

Reply:
ok, better to tag it with v0.6.x-rc0 for these untested pre-releases. If the original v0.6.3 was removed and it's not used anywhere, and this just modifies its scope, it's probably ok to release it... Otherwise, it's confusing why 0.6.3 is skipped.
Alternatively, perhaps we can re-add the original v0.6.3 tag, but have the release empty without binaries and the release notes would say that it was yanked and link v0.6.4.